### PR TITLE
KOGITO-3128 Quarkus dev mode breaks when adding a new DMN file

### PR DESCRIPTION
**Pull request by tarilabs**

### CODE REVIEW

File: kogito-quarkus-extension/deployment/src/main/java/org/kie/kogito/quarkus/deployment/KogitoAssetsProcessor.java [hunk 1]
```diff
@@ -1,5 +1,38 @@
+/*
```
[the remark this answers is not in the document]
Comment (tarilabs) [reply kogito-quarkus-extension/deployment/src/main/java/org/kie/kogito/quarkus/deployment/KogitoAssetsProcessor.java]: done with e37d526

**Review by danielezonca — APPROVED** [review]
Thanks for the PR, approving :+1:

**Comment by tarilabs** [comment]
thank you @danielezonca and @mariofusco for the support on this topic, I believe we uncovered quite some gaps in the design too

**Comment by danielezonca** [comment]
jenkins retest this please

**Comment by tarilabs** [comment]
Failings relate to `codegen` which this PR doesn't even touch 🤷‍♂️ 

```
New failing - 34
testProcessGeneration{String}[92] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[93] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[94] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[95] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[96] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[97] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[98] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[99] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[100] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[101] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[102] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[103] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[104] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[105] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[106] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[107] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[108] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[109] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[110] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[111] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[112] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[113] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[114] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[115] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[116] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[117] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[118] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[119] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[120] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[121] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[122] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[123] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[124] – org.kie.kogito.codegen.process.ProcessGenerationTest
<1s
testProcessGeneration{String}[125] – org.kie.kogito.codegen.process.ProcessGenerationTest
```

**Comment by tarilabs** [comment]
Jenkins test again

**Comment by tarilabs** [comment]
jenkins retest this please

**Review by hellowdan — APPROVED** [review]

**Pull request merged**